src/scripts/cfo.zig

```diff
@@ -54,8 +59,13 @@ const stdx = @import("../stdx.zig");
 const Shell = @import("../shell.zig");
 
 pub const CLIArgs = struct {
-    budget_minutes: u64 = 10,
-    hang_minutes: u64 = 30,
+    /// How long to run the cfo before exiting (so that cfo_supervisor can refresh our code).
+    budget_minutes: u64 = 40,
+    /// The interval for flushing accumulated seeds to the devhub.
+    /// In addition to this interval, any remaining seeds will be uploaded at the end of the budget.
+    refresh_minutes: u64 = 5,
+    /// A fuzzer which takes longer than this timeout is killed and counts as a failure.
+    timeout_minutes: u64 = 30,
```

Comment:
check that budget_minutes >= timeout_minutes. 

refresh_minutes I think we don't need to check, because we always have upload at the end. Though perhaps assert that it is at least one minute?